test-configs.yaml

```
@@ -183,27 +183,47 @@ test_plans:
     params:
       job_timeout: '120'
 
-  ltp-crypto:
+  ltp-opts: &ltp-rootfs
     rootfs: debian_buster-ltp_nfs
     pattern: 'ltp/{category}-{method}-{protocol}-{rootfs}-ltp-template.jinja2'
     filters:
       - blocklist: *kselftest_defconfig_filter
-    params:
+    params: &ltp-params
       skip_install: "true"
-      tst_cmdfiles: "crypto"
+      tst_cmdfiles: ""
       job_timeout: '15'
       skipfile: skipfile-lkft.yaml
 
+  ltp-crypto:
+    <<: *ltp-rootfs
+    params:
+      <<: *ltp-params
+      tst_cmdfiles: "crypto"
+
   ltp-ipc:
-    rootfs: debian_buster-ltp_nfs
-    pattern: 'ltp/{category}-{method}-{protocol}-{rootfs}-ltp-template.jinja2'
-    filters:
-      - blocklist: *kselftest_defconfig_filter
+    <<: *ltp-rootfs
     params:
-      skip_install: "true"
+      <<: *ltp-params
       tst_cmdfiles: "ipc"
-      job_timeout: '15'
-      skipfile: skipfile-lkft.yaml
+
+  ltp-lib:
+    <<: *ltp-rootfs
+    params:
+      <<: *ltp-params
+      tst_cmdfiles: "lib"
+
+  ltp-mm:
+    <<: *ltp-rootfs
+    params:
+      <<: *ltp-params
+      tst_cmdfiles: "mm"
+      job_timeout: '30'
```

Patch number 2, to add `ltp-mm`.

```suggestion
  ltp-mm:
    <<: *ltp
    params:
      <<: *ltp-params
      tst_cmdfiles: "mm"
      job_timeout: '30'
```